clients/apps/web/src/app/embed/product.svg/route.tsx

```diff
@@ -0,0 +1,126 @@
+import { ProductCard } from '@/components/Embed/ProductCard'
+import { getServerURL } from '@/utils/api'
+import {
+  Product,
+} from '@polar-sh/sdk'
+const { default: satori } = require('satori')
+
+export const runtime = 'edge'
+
+const getProduct = async (
+  productId: string,
+  productPriceId: string,
+): Promise<Product | undefined> => {
+  let url = getServerURL(`/v1/products/${productId}`)
```

Comment:
This won't work anymore with the API changes. You should use the Storefront API, similar to here: 

https://github.com/polarsource/polar/blob/a13aee8f95f2354ee764d631bdbf5d490e46d446/clients/apps/web/src/app/embed/tiers.svg/route.tsx#L13-L28

Reply:
Noticing that we only have access to the product ID here.

Could probably be nice then to introduce an `/embed` API with different endpoints to return the data we need to render each type of embed. I could do that fairly quickly.

Reply:
Thank you @frankie567. Nice catch. Didn't discover it in my branch of course given we did these two things in parallel 😅 I'lll fix this today. Agree re: `/embed` (internal) API being a nice solution and would also offer the ability to do some caching in a better/cleaner way over time which would be nice.